packages/backend/src/services/DashboardService/DashboardService.ts

```diff
@@ -125,6 +87,23 @@ export class DashboardService {
         };
     }
 
+    async hasDashboardSpaceAccess(
```

Comment:
any reason to move `hasDashboardSpaceAccess` and `checkUpdateAccess` ?

Reply:
I only added the projectUuids here, the re-formatting is probably ESLint or prettier